src/app/group/timeline/timeline.component.ts

```diff
@@ -9,9 +9,10 @@ import { TASKS } from './mock-tasks'
 })
 
 export class TimelineComponent implements OnInit {
-
   tasks = TASKS;
   selectedTask: Task;
+  selectedInfo: Task;
+  tabs = 'Hello';
```

Comment:
what is this variable needed (tabs) for?

Reply:
They were unused, thanks.
Fixed 👍